lib/url.c

```diff
@@ -4010,7 +4010,12 @@ static void fix_hostname(struct connectdata *conn, struct hostname *host)
 #ifdef USE_LIBIDN2
     if(idn2_check_version(IDN2_VERSION)) {
       char *ace_hostname = NULL;
-      int rc = idn2_lookup_ul((const char *)host->name, &ace_hostname, 0);
+#ifdef IDN2_TRANSITIONAL
+      int flags = IDN2_INPUT_NFC | IDN2_TRANSITIONAL;
```

Comment:
I was a bit too quick. Which version adds `IDN2_INPUT_NFC`? It seems we need to check for that as well, right?